modules/monitoring-understanding-metrics.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-metrics.adoc
+
+[id="understanding-metrics_{context}"]
+= Understanding metrics
+
+In {product-title} {product-version}, cluster components are monitored by scraping metrics exposed through service endpoints. You can also configure metrics collection for user-defined projects. Metrics enable you to monitor how cluster components and your own workloads are performing.
+
+You can define the metrics that you want to provide for your own workloads by using Prometheus client libraries at the application level.
+
+In {product-title}, metrics are exposed through an HTTP service endpoint under the `/metrics` canonical name. You can list all available metrics for a service by running a curl query against `http://<route>/metrics`. For instance, you can expose a route to the `prometheus-example-app` example application and then run the following to view all of its available metrics:
```

Comment:
`http://<route>/metrics`
should be
`http:///<endpoint>/metrics`

Reply:
Also add a `\` so that it doesn't render an actual link:

```
\http:///<endpoint>/metrics
```

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#example-urls

Reply:
Consider possibly marking up `curl` in the sentence as well?

Reply:
Thank you for your suggestions. I have now resolved each one.